scripts/generate-AUTHORS.py

```diff
@@ -0,0 +1,199 @@
+
+	# ...What?
+	"Netbios Domain Administrator <admin@gameserver2.(none)>",
+
+	# Aliases of Mikkel Krautz <mikkel@krautz.dk>
+	"Mikkel <mikkel@krautz.dk>",
+	"Mikkel Krautz <mkrautz@users.sourceforge.net>",
+
+	# Alias of tkmorris <mauricioarozi@gmail.com>
+	"morris <tkmorris@users.sourceforge.net>",
+
+	# Alias of bendem <online@bendem.be>
+	"bendem <bendembd@gmail.com>",
+
+	# Alias of arrai <array.of.intellect@gmail.com>
+	"Arrai <arrai@users.sourceforge.net>",
+
+	# Alias of Joël Troch <joel.troch62@gmail.com>
+	"Joël Troch <https://github.com/JoelTroch>",
+
+	# Matthias Vogelgesang <matthias.vogelgesang@gmail.com>
+	"Matthias Vogelgesang <m0ta@users.sourceforge.net>",
+
+	# Alias of Tuck Therebelos <snares@users.sourceforge.net>
+	"Snares <snares@users.sourceforge.net>",
+
+	# Aliases of Natenom <natenom@natenom.com>
+	"Natenom <natenom@googlemail.com>",
+	"Natenom <natenom@natenom.name>",
+
+	# Alias of Arcenciel <arcenciel@users.sourceforge.net>
+	"arcenciel <arcenciel@users.sourceforge.net>",
+
+	# Alias of Jonathan E. Hansen <zentriple@users.sourceforge.net>
+	"Jonathan <zentriple@users.sourceforge.net>",
+	"zentriple <zentriple@users.sourceforge.net>",
+	"Zentriple <zentriple@users.sourceforge.net>",
+
+
+	# Alias of Patrick Matthäi <pmatthaei@debian.org>
+	"Patrick Matthäi <the-me88@users.sourceforge.net>",
+
+	# Alias of Jan Klass <kissaki@gmx.de>
+	"Jan Klass <kissaki0@gmail.com>",
+
+	# Alias of Necromancer <necro3@users.sourceforge.net>
+	"Necromancer <necromancer3@users.sourceforge.net>",
+
+	# Alias of Svenne33 <svenne33@users.sourceforge.net>
+	"svenne33 <svenne33@users.sourceforge.net>",
+
+	# Potentially an alias of BAYSSE Laurent <lolo_32@users.sourceforge.net>
+	"lolo_32 <Alex@.(none)>",
+)
+
+def gitAuthorsOutput():
+	p = subprocess.Popen(["git", "log", "--all", "--format=%aN <%aE>"], stdout=subprocess.PIPE)
+	stdout, stderr = p.communicate()
+	if stdout is not None:
+		stdout = stdout.decode('utf-8')
+	if stderr is not None:
+		stderr = stderr.decode('utf-8')
+	if p.returncode != 0:
+		raise Exception('"git log" failed: %s', stderr)
+	return stdout
+
+def main():
+	locale.setlocale(locale.LC_ALL, "")
+
+	authorsSet = set()
+	authorsText = gitAuthorsOutput()
+	for line in authorsText.split("\n"):
+		if line == '':
+			continue
+
+		if line == "zapman <unknown>":
+			line = "zapman"
+
+		# Fix "=?UTF-8 Michał Żukowski?=" and possibly others like it.
+		if line.startswith("=?UTF-8 "):
+			line = line.replace("=?UTF-8 ", "")
+			line = line.replace("?=", "")
+
+		# Use GitHub URL instead of $user@users.noreply.github.com
+		if '@users.noreply.github.com' in line:
+			line = line.replace('@users.noreply.github.com', '')
+			line = line.replace('<', '<https://github.com/')
+
+		if line in blacklist:
+			continue
+
+		authorsSet.add(line)
+
+	f = codecs.open("AUTHORS", "w", "utf-8")
+	f.write('''// This is the official list of people who have contributed
+// to, and hold the copyright to Mumble.
```

Comment:
"and/or hold the copyright" maybe? IANAL but afaik not every contribution gives you copyright and not all who hold some copyright do so because they contributed code (e.g. the bonjour code we built on).

Or maybe just strike the copyright part completely. Dunno...